### PR TITLE
update readme for Ember Octane

### PR DESCRIPTION
**Pull request by jelhan**
Updates the readme for Ember Octane. In detail it does:

- Refactor curly bracket component invocation to angle bracket syntax.
- It remove duplicated examples (curly + angle bracket). It's not necessary anymore in my opinion as there isn't anything special about conversion for this addon. It does neither use positional params in curly invocation nor require special arguments for the mapping to HTML attributes.
- Use `{{on}}` element modifier for event binding instead of `{{action}}`.
- Uses native classes in the example.
- Use `@action` decorator instead of `actions` hash in the examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,52 +90,33 @@ this.notify.alert('Unique Message', { id: 'some-unique-identifier' });
...
 The others you will need to provide a `source` property, so secondary containers should be used as follows:
 
-```hbs
-{{ember-notify source=someProperty}}
-```
-
-Using angle bracket invocation:
-
 ```hbs
 <EmberNotify @source={{someProperty}} />
```
Comment (jelhan) [inline README.md]: Haven't ever used that feature but if I got it right, this should be `{{this.alternativeNotify}}` to match the component JavaScript example, isn't it?

```suggestion
 <EmberNotify @source={{this.alternativeNotify}} />
```

File: README.md [hunk 2]
```diff
@@ -152,24 +127,15 @@ Using angle bracket invocation:
 
 Two arguments are passed to the block: `message` object, and `close` action. Make sure
 you are using *Closure Actions* syntax passing the action (e. g. `<a {{action close}}` or
-`{{your-component close=(action close)}}`.
+`<YourComponent @close={{action close}} />`.
```
Comment (jelhan) [inline README.md]: `this` binding should be done internally using `@action` decorator to drop that requirement and simplify to `<YourComponent @close={{close}} />`. But will leave that one for another PR.

File: README.md [hunk 3]
```diff
@@ -9,53 +9,47 @@
...
-
 ```handlebars
 <EmberNotify @closeAfter={{4000}} />
 ```
 
 Or you can control when each message is closed:
 
 ```js
-let message = this.notify.alert(`You can control how long it's displayed`, {
+let message = this.notify.alert('You can control how long it's displayed', {
```
Comment (btecu) [inline README.md]: This is not valid `js` because there's a `i` in the sentence

File: README.md (hunk 1 above)
Comment (btecu) [reply README.md]: Yes, you got it right

**Comment by btecu** [comment]
Thanks! Overall looks good. I'll wait until Octane is released before merging.

File: README.md (hunk 3 above)
Comment (jelhan) [reply README.md]:
```suggestion
let message = this.notify.alert('You can control how long it is displayed', {
```

**Comment by jelhan** [comment]
@btecu Thanks for your review! Addressed the comments. Do you want me to squash or will you squash it when you merge the PR anyway?

**Comment by btecu** [comment]
@jelhan I can squash when merging.

**Comment by btecu** [comment]
Thanks @jelhan!
